repo/packages/K/kafka/500/resource.json

```diff
@@ -0,0 +1,62 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz",
+      "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.55.2/bootstrap.zip",
+      "kafka-tgz": "https://downloads.mesosphere.com/kafka/assets/kafka_2.12-1.1.1.tgz",
```

Comment:
How do we check all the URI's under this section are pointing to the latest tar and zip files?

Reply:
@wavesoft @shubhanilBag @vishnu2kmohan ^^^

Reply:
I'm not sure I understand your concern @imaxxs

Reply:
When we soak the package these are implicitly tested.

Reply:
Technically, we should bump the JRE to 8u202 and libmesos-bundle to 1.12.2 or 1.12.3 but we can do that for the next release.